target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -87,6 +87,7 @@ get_status_led() {
 	ht-tm02)
 		status_led="$board:blue:wlan"
 		;;
+	afoundry-ew1200|\
```

Comment:
Please, follow the general rule and keep all boards in alphabetical order here (in this example, you should move whole block above the one starting with `ai-br100`) and in other scripts your patch changes, don't forget about that after you change the board name (we don't use vendor name as prefix in board name).